wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/devfile/convert/DevfileConverter.java

```diff
@@ -180,12 +180,12 @@ public WorkspaceConfigImpl devFileToWorkspaceConfig(
   }
 
   private static void validateCurrentVersion(Devfile devFile) throws DevfileFormatException {
-    if (Strings.isNullOrEmpty(devFile.getSpecVersion())) {
+    if (Strings.isNullOrEmpty(devFile.getApiVersion())) {
       throw new DevfileFormatException("Provided Devfile has no spec version specified");
     }
-    if (!CURRENT_SPEC_VERSION.equals(devFile.getSpecVersion())) {
+    if (!CURRENT_API_VERSION.equals(devFile.getApiVersion())) {
       throw new DevfileFormatException(
-          format("Provided Devfile has unsupported version '%s'", devFile.getSpecVersion()));
+          format("Provided Devfile has unsupported version '%s'", devFile.getApiVersion()));
```

Comment:
It's not directly related to your changes, but please consider adding the supported version in error message.